include/lsst/jointcal/StarList.cc

```diff
@@ -18,138 +18,6 @@ namespace lsst {
 namespace jointcal {
```

Comment:
Nobody expects to include a `.cc` file; consider either renaming (if it must be used as a header file, which I assume is the case) or moving it to `src/`.